source/includes/markdown/_17-audit-log.html.md

```diff
@@ -38,26 +38,27 @@ All user events operate on the _User_ resource type.
  
 ### Admin Settings
  
-All admin settings events operate on the _Workspace_ resource type.
```

Comment:
@androse  If we keep the table at just two columns, maybe we can re-include this, and just note that `workspace_announcement_created` and `workspace_announcement_removed` operate on different resources as well.

Reply:
Sounds like a reasonable compromise.